crates/holochain_keystore/src/test_keystore.rs

```diff
@@ -83,7 +83,7 @@ pub async fn spawn_test_keystore() -> LairResult<MetaLairClient> {
 pub fn test_keystore() -> MetaLairClient {
     holochain_util::tokio_helper::block_on(
         async move { spawn_test_keystore().await.unwrap() },
-        std::time::Duration::from_secs(1),
+        std::time::Duration::from_secs(5),
```

Comment:
When multiple tests run this in parallel, 1 seconds causes a timeout many times.